From: Nathan Fontenot <nfont@austin.ibm.com>
Subject: [PATCH] memory hotplug: Define memory_block_size_bytes for x86_64 with CONFIG_X86_UV
Patch-mainline: Queued in driver-core-2.6.git/driver-core-next branch
References: bnc#643266

From 1dc41aa6d6172d61c10638d11933a3595a41d51a Mon Sep 17 00:00:00 2001
From: Nathan Fontenot <nfont@austin.ibm.com>
Date: Thu, 20 Jan 2011 10:46:15 -0600
Subject: [PATCH] memory hotplug: Define memory_block_size_bytes for x86_64 with CONFIG_X86_UV

Define a version of memory_block_size_bytes for x86_64 when CONFIG_X86_UV is
set.

Signed-off-by: Robin Holt <holt@sgi.com>
Signed-off-by: Jack Steiner <steiner@sgi.com>
Signed-off-by: Nathan Fontenot <nfont@austin.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
Signed-off-by: Nikanth Karthikesan <knikanth@suse.de>

---

Index: linux-2.6.32-SLE11-SP1/arch/x86/mm/init_64.c
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/arch/x86/mm/init_64.c
+++ linux-2.6.32-SLE11-SP1/arch/x86/mm/init_64.c
@@ -50,6 +50,7 @@
 #include <asm/cacheflush.h>
 #include <asm/init.h>
 #include <linux/bootmem.h>
+#include <asm/uv/uv.h>
 
 static unsigned long dma_reserve __initdata;
 
